introduction.md

```diff
@@ -0,0 +1,3 @@
+Codeschool Mastering Github course, dojo_rules fork of Xoiiu
```

Comment:
Looks good! Could you also please mention your favorite Code School path in your introduction? This could be Ruby, JavaScript, HTML & CSS, iOS or Electives.